recipes/pywinauto/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pywinauto" %}
+{% set version = "0.6.5" %}
+{% set file_ext = "zip" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  sha256: e115463806acc41b8b5aeaeaece9cd5996a3091d1baa18b1dc91736c7684449d
+
+build:
+  number: 0
+  skip: True  # [not win]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
```

Comment:
It looks like there are different requirements based on the platform:

https://github.com/pywinauto/pywinauto/blob/master/setup.py#L68

Reply:
I've added them with selectors.